e2e/homepage.spec.ts

```diff
@@ -2,7 +2,7 @@ import { expect, test } from '@playwright/test'
 import { testAuth } from './fixtures/authenticatedUserPages'
 import { waitForInitialPageLoad } from './helpers/utils'
 
-test.describe('Homepage', () => {
+test.describe('Homepage - Unauthenticated', () => {
```

Comment:
Separate these tests into different suites, so the unauthenticated suite can still be run without setting up the authenticated user page fixtures